components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/ExceptionCodes.java

```diff
@@ -261,6 +261,8 @@ public enum ExceptionCodes implements ErrorHandler {
     INVALID_OAS3_FOUND(900762, "Invalid OpenAPI V3 definition found", 400, "Invalid OpenAPI V3 definition found"),
     NO_RESOURCES_FOUND(900763, "No resources found", 404, "API must have at least one resource defined"),
     ERROR_REMOVING_EXAMPLES(900764, "Internal Error While Processing Swagger Definition", 500, "Couldn't remove one or more examples from the swagger definition"),
+    INVALID_THROTTLING_LIMIT_FOUND(900765, "Invalid Throttling Limit Found", 400,
+            "Invalid Throttling Limit Found under x-throttling-limit extension"),
```

Comment:
single line

Reply:
nothing to change as it should go to two lines considering the checkstyle guideline we follow